lib/core/nuxt.js

```diff
@@ -136,23 +136,41 @@ module.exports = class Nuxt {
     })
   }
 
-  listen(port = 3000, host = 'localhost') {
+  listen(port = 3000, host = 'localhost', unix = undefined) {
     return new Promise((resolve, reject) => {
+      const args = { exclusive: false }
+
+      if (unix) {
+        args.path = unix
+      } else {
+        args.port = port
+        args.host = host
+      }
+
       const server = this.renderer.app.listen(
-        { port, host, exclusive: false },
+        args,
         err => {
           /* istanbul ignore if */
           if (err) {
             return reject(err)
           }
 
-          const _host = host === '0.0.0.0' ? 'localhost' : host
-          // eslint-disable-next-line no-console
-          console.log(
-            '\n' +
-              chalk.bgGreen.black(' OPEN ') +
-              chalk.green(` http://${_host}:${port}\n`)
-          )
+          if (!unix) {
+            const _host = host === '0.0.0.0' ? 'localhost' : host
+            // eslint-disable-next-line no-console
+            console.log(
```

Comment:
Please use [consola](https://github.com/nuxt/consola) for these calls.

Reply:
:b:onzola